### ISSUE TITLE
More (minimalistic) appearance options

### ISSUE THREAD

**Issue by BerberBerthold** [krawieck/lemmur#302]
I love Lemmy, and I want to start using it more.

The Lemmur app is pretty awesome, but unfortunately, it is way too cluttered for my taste.  There sadly aren't many appearance customization options.  Ideally, I would want something that is similar to RedReader, and Android client for Reddit.  I just love all of the customization options.

I wish for the following options most:
- Be able to disable big previews (images and self post contents) 
- Be able to remove remove the large bar at the bottom
- Be able to remove the large bar at the top
- When big previews are enabled, be able to enable small thumbnails that don't take up more vertical space than the headline
- Font size
- Be able to remove upvote downvote etc buttons

Basically, I want to be able to see a concise list of posts, instead of having an instagram/new reddit style viewing experience where a single previewed post can take up the whole screen.

**Comment by shilangyu** [comment]
While it's not planned for the near future I would definitely like these options as well
